src/main/java/seedu/tp/flashcard/Flashcard.java

```diff
@@ -52,4 +54,35 @@ public void setReviewStatus (boolean isReviewed) {
     public String getReviewIcon() {
         return (isReviewed ? "Y" : "N");
     }
+
+    /**
+     * Sets the flashcard's priority level.
+     *
+     * @param priorityLevel priority level to be set
+     */
+    public void setPriorityLevel(int priorityLevel) {
+        this.priorityLevel = priorityLevel;
+    }
+
+    /**
+     * Returns the number of "*"s based on flashcard's priority level.
+     *
+     * @return "*"s to indicate priority level
+     */
+    public String getPriority() {
```

Comment:
Maybe would be better idea to rename this to getPriorityAsString or something like that?

Reply:
Edited!